cmake/Modules/TestDatasets.cmake

```diff
@@ -0,0 +1,39 @@
+## csv, rdg test inputs
+set(TEST_DATASETS "${PROJECT_SOURCE_DIR}/external/test-datasets")
+set(RDG_TEST_DATASETS "${PROJECT_SOURCE_DIR}/external/test-datasets/rdg_datasets")
+set(CSV_TEST_DATASETS "${PROJECT_SOURCE_DIR}/external/test-datasets/csv_datasets")
+set(MISC_TEST_DATASETS "${PROJECT_SOURCE_DIR}/external/test-datasets/misc_datasets")
+
+## latest supported rdg storage_format_version
+#TODO(emcginnis) get this envar in RDGPartHeader.h instead of having to hard code it here and there
```

Comment:
If this value isn't needed until test time (which I assume is the case), you could just have a tiny C++ program that prints the value you need that is built, but not installed.